experiments/browser_based_querying/README.md

```diff
@@ -9,8 +9,12 @@ See the `example_queries` directory for some example queries to run. Live demo h
 ## Development
 
 - Install the version of node and `npm`.
```

Comment:
Do people need to install `pnpm` separately?

Reply:
No, unless they installed node from homebrew: https://pnpm.io/installation#using-corepack

Reply:
Saying the unless you installed node from homebrew part feels like an implementation detail that can change later

Reply:
It would be good to add a note for homebrew. We don't wanna have to debug broken workflows later if we already know a possible issue that might come up.